rust/yarp-sys/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "yarp-sys"
+version = "0.1.0"
+edition = "2021"
+license-file = "../../LICENSE.md"
```

Comment:
is it possible to add a `license` field with an SPDX expression so automated tools can parse the license? `license-file` is meant to be an escape hatch when it is otherwise impossible to specify the license via SPDX.

Reply:
This makes sense to me, but after looking at the `LICENSE.md` file, I don't know what to populate `license` with...